src/main/scala/temple/builder/project/ProjectConfig.scala

```diff
@@ -9,12 +10,13 @@ object ProjectConfig {
     override def toString: String = image + ":" + version
   }
 
-  val defaultLanguage: ServiceLanguage = ServiceLanguage.Go
-  val defaultDatabase: Database        = Database.Postgres
-  val defaultAuth: ServiceAuth         = ServiceAuth.Email
-  val authPort: Int                    = 1024
-  val authMetricPort: Int              = 1025
-  val serviceStartPort: Int            = 1026
+  val defaultLanguage: ServiceLanguage  = ServiceLanguage.Go
+  val defaultDatabase: Database         = Database.Postgres
+  val defaultAuth: ServiceAuth          = ServiceAuth.Email
+  val defaultPreparedType: PreparedType = PreparedType.DollarNumbers
```

Comment:
Shouldn’t this be a function of the language?

Reply:
Good point, I'll fix